src/components/ScorecardPickerDialog.tsx

```diff
@@ -388,7 +414,7 @@ const ScorecardPickerDialog: React.FC<ScorecardTemplatesDialogProps> = ({
             >
                 {/* Header */}
                 <div className="flex justify-between items-center p-4">
-                    <h2 className="text-white text-lg font-normal">New scorecard</h2>
+                    <h2 className="text-white text-lg font-normal">{type === 'assignment' ? 'New key areas' : 'New scorecard'}</h2>
```

Comment:
we can just call them scorecard only - no need to call key areas - will be very confusing for people